packages/block-library/src/archives/style.scss

```diff
@@ -0,0 +1,3 @@
+.wp-block-archives-dropdown {
```

Comment:
This class is never actually assigned to the `select` on the front-end, so it's not applied.

This note applies to the categories block as well.

Reply:
Hmm I thought these were being added with the below code:

`$class = "wp-block-categories wp-block-categories-{$type}";`

I'll take a look and see what I find out

Reply:
It looks like it is being applied but it has a low specificity compared a bunch of other selectors.